### PR TITLE
add chinese translate to desktop/locales and web/locales

### PR DESCRIPTION
**Pull request by jobinson99**
### All Submissions:

* [ ] Have you followed the guidelines in our Contributing document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [ ] Does your submission pass tests?

> Note: please do NOT include build files (those generate by the build-xxx commands) with your PR,

Thank you for your help in advance, much appreciated !

### CODE REVIEW

File: packages/desktop/locales/en.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "english":"english",
   "german":"german",
   "french":"french",
+  "chinese":"Chinese",
```
Comment (platypii) [inline packages/desktop/locales/en.json:6]: language should be lowercase to match other languages

File: packages/web/locales/zh.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  "top":"上",
+  "bottom":"下",
+  "left":"左",
+  "right":"右",
+
+  "setProjectionType":"视图类型",
+  "perspective":"透视",
+  "orthographic":"正视",
+
+  "editorCommand":"Editor command",
```
Comment (z3dev) [inline packages/web/locales/zh.json:58]: Hmm... I would guess that an equivalent Chinese phrase exists.

**Review by z3dev — CHANGES_REQUESTED** [review]

**Review by z3dev — CHANGES_REQUESTED** [review]
@jobinson99 Thanks for these nice additions. There are a few small requests.

**Review by z3dev — APPROVED** [review]
@jobinson99 SUPER! Thanks for making those changes. I hope this builds more interest from the folks in China (and other countries)

P.S. You can 'resolve' any requests after making changes. 😃

**Review by z3dev — APPROVED** [review]
@jobinson99 thanks for the updates. this looks fine now.

**Pull request merged**